src/LoggingService.ts

```diff
@@ -0,0 +1,75 @@
+import HazelcastClient = require('./HazelcastClient');
+import DefaultLogger = require('./DefaultLogger');
+import NoLogger = require('./NoLogger');
+export enum LogLevel {
+    ERROR = 0,
+    WARN = 1,
+    INFO = 2,
+    DEBUG = 3,
+    TRACE = 4
+}
```

Comment:
isn't it same thing as TRACE in our case? When you set to TRACE it should show all logs as there is no other log level bigger than TRACE.